config/environment.js

```diff
@@ -41,6 +41,12 @@ module.exports = function (environment) {
         discordLink: 'https://discord.com/channels/480462759797063690/486235962700201984'
       },
       sourceRepo: 'https://github.com/DazzlingFugu/ember-fr-guides-source',
+      texts: {
+        searchInputPlaceholder: 'Rechercher dans le Guide',
+        searchResultsPlaceholder: 'Résultats de la recherche',
+        currentChapter: 'Nous avons couvert',
```

Comment:
Are we sure about this translation?
Shouldn't it be `Chapitre actuel` instead? To match the following `nextChapter: 'Chapitre suivant'`.

Reply:
It's an accurate translation, but I admit the English is a bit verbose for my taste, especially as a "next button" text 🤔

Reply:
I think I need more clarification because I don't get the "the English is a bit verbose", assuming you are talking about "next chapter / chapitre suivant" ... ? What were you talking about?

When reading any guide, it's more than natural to see "next chapter / chapitre suivant" on a button that, well, actually brings us to the next page of the guide/current section we are reading.

But I don't see where this is used in the guides so I may be missing some context.